CMakeLists.txt

```diff
@@ -188,9 +188,12 @@ if(KOKKOSTOOLS_HAS_VARIORUM)
 endif()
 
 # GPU profilers
-if(Kokkos_ENABLE_CUDA)
+if(KOKKOSTOOLS_HAS_NVTX)
   add_subdirectory(profiling/nvtx-connector)
   add_subdirectory(profiling/nvtx-focused-connector)
+else()
+  message(STATUS "Skipping nvtx-connector:         NVTX is not enabled (maybe Kokkos was not found, or not configured with the CUDA backend)")
+  message(STATUS "Skipping nvtx-focused-connector: NVTX is not enabled (maybe Kokkos was not found, or not configured with the CUDA backend)")
```

Comment:
Although I agree there is room for improvement, the changes suggested are somewhat shortsighted as they introduce an inconsistency with the roctx detection logic right below.